app/src/main/res/values/gdg_themes.xml

```diff
@@ -42,11 +44,32 @@
     <item name="windowActionBar">false</item>
   </style>
   
-  <style name="Theme.GDG.Main">
+  <style name="Theme.GDG.Main" parent="Theme.GDG.WithNavDrawer">
     <item name="actionBarInsetStart">@dimen/keyline_2_minus_16dp</item>
   </style>
 
-  <style name="Theme.GDG.Special.Wtm" parent="Theme.GDG">
+  <!-- Special Event Series-->
+  <style name="Theme.GDG.Special" parent="Theme.GDG.WithNavDrawer"/>
+
+  <!-- Experts Directory -->
+  <style name="Theme.GDG.GDE" parent="Theme.GDG.WithNavDrawer"/>
```

Comment:
I thought about it really much and I ended up doing it because Google IO app did it like this. They have empty styles for different activities. I also did that to get your opinion.